package.json

```diff
@@ -3,12 +3,20 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "postinstall": "cd packages/idyll-compiler && npm run build && cd ../idyll-components && npm run build && cd ../idyll-document && npm run build"
+    "postinstall": "cd packages/idyll-compiler && npm run build && cd ../idyll-components && npm run build && cd ../idyll-document && npm run build && opencollective postinstall"
   },
   "devDependencies": {
     "lerna": "^2.0.0"
   },
   "workspaces": [
     "packages/*"
-  ]
-}
+  ],
+  "dependencies": {
+    "opencollective": "^1.0.3"
+  },
```

Comment:
Let's remove the dependency and post-install hook for now. Since we're structured as a monorepo only contributing devs would see this here anyway